src/assets/data/edition/series/1/section/5/op12/source-description.json

```diff
@@ -1592,8 +1592,298 @@
+                        },
+                        {
+                            "svgGroupId": "TODO",
+                            "measure": "22 <br /> bis 23",
+                            "system": "Klav. o.",
+                            "position": "2/4",
+                            "comment": "Gis<sub>1</sub>/Cis geändert zu As<sub>1</sub>/Des."
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "id": "source_Cb_corr",
+            "label": "Korrekturen in <strong>C<sup>b</sup></strong>",
+            "description": [
+                "<strong>C<sup>b</sup></strong> weist zwei voneinander unterscheidbare Korrekturschichten auf: mit Tinte ggf. auf Rasur (Korrekturen 1) und mit Bleistift (Korrekturen 2). Der Zustand vor und nach Korrekturen 1 ist nicht eindeutig entzifferbar. Korrekturen 2 betreffen die Überarbeitung zu der in <strong>F</strong> etablierten Textfassung von <em>Die geheimnisvolle Flöte</em> M 217. Auf eine tabellarische Darstellung der beiden Korrekturschichten wird verzichtet, da sie auf Grund der zahlreichen Abweichungen der Grundschicht zu der edierten einzigen Textfassung nicht praktikabel ist."
+            ],
+            "comments": [ ]
+        },
+        {
+            "id": "source_Cc_corr",
+            "label": "Korrekturen in <strong>C<sup>c</sup></strong>",
+            "description": [
+                "<strong>C<sup>c</sup></strong> weist zwei voneinander unterscheidbare Korrekturschichten auf: mit Tinte ggf. auf Rasur bzw. auf Tektur (Korrekturen 1) und mit Bleistift (Korrekturen 2). Der Zustand vor und nach Korrekturen 1 ist nicht eindeutig entzifferbar. Korrekturen 2 betreffen die Überarbeitung zu der in <strong>F</strong> etablierten Textfassung von „Schien mir’s, als ich sah die Sonne“ M 213. Auf eine tabellarische Darstellung der beiden Korrekturschichten wird verzichtet, da sie auf Grund der zahlreichen Abweichungen der Grundschicht zu der edierten einzigen Textfassung nicht praktikabel ist."
+            ],
+            "comments": [ ]
+        },
+        {
+            "id": "source_Cd_corr",
+            "label": "Korrekturen in <strong>C<sup>d</sup></strong>",
+            "description": [
+                "<strong>C<sup>d</sup></strong> weist drei voneinander unterscheidbare Korrekturschichten auf: mit Tinte ggf. auf Rasur (Korrekturen 1), mit Bleistift (Korrekturen 2) und mit Kopierstift (Korrekturen 3). Der Zustand vor und nach Korrekturen 1 ist nicht eindeutig entzifferbar, post correcturam entspricht er in weiten Teilen Textfassung 1 von <em>Gleich und Gleich</em> („Ein Blumenglöckchen“) M 216. Korrekturen 2 und 3 betreffen die Überarbeitung zu der in <strong>F</strong> etablierten Textfassung 2. Auf eine tabellarische Darstellung der beiden Korrekturschichten wird verzichtet, da sie auf Grund der zahlreichen Abweichungen der Grundschicht zu der edierten einzigen Textfassung nicht praktikabel ist."
```

Comment:
"zu der edierten einzigen Textfassung" vs. "in F etablierten Textfassung 2?

Reply:
Danke! Wird geändert.